src/main.cpp

```diff
@@ -7769,11 +7763,11 @@ int64_t GetProofOfStakeReward(int nHeight, int64_t nCoinAge, int64_t nFees)
     int64_t nRewardCoinYear;
     nRewardCoinYear = MAX_MINT_PROOF_OF_STAKE;
 
-    if(nHeight < 7 * DAILY_BLOCKCOUNT)
+    if(nHeight-1 < 7 * DAILY_BLOCKCOUNT)
```

Comment:
It is like this in NavCoin original code.

https://github.com/navcoindev/navcoin2/blob/master/src/main.cpp Line 1337